apps/meteor/app/livechat/server/api/v1/visitor.ts

```diff
@@ -79,28 +80,32 @@ API.v1.addRoute(
 			);
 
 			if (customFields && Array.isArray(customFields) && customFields.length > 0) {
-				const keys = customFields.map((field) => field.key);
 				const errors: string[] = [];
+				const keys = customFields.map((field) => field.key);
+
+				const livechatCustomFields = await LivechatCustomField.findByScope(
```

Comment:
Can u add an api test to ensure this won't break anything? one that is: if no custom fields are received, then the required validation shouldn't apply